debug/p2p/p2pDebugger.go

```diff
@@ -0,0 +1,213 @@
+package p2p
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"sync"
+	"time"
+
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/display"
+)
+
+var log = logger.GetOrCreate("debug/p2p")
+
+const printTimeOneSecond = time.Second //if this needs to be changed, remember to divide the values when computing metrics
```

Comment:
add a const for X * time.second - use X for division :)

Reply:
should have never wrote that comment :))))

Reply:
done